imports/wasi_snapshot_preview1/wasi_test.go

```diff
@@ -50,8 +52,8 @@ func TestNewFunctionExporter(t *testing.T) {
 		defer r.Close(testCtx)
 
 		// Export the default WASI functions
-		wasiBuilder := r.NewHostModuleBuilder(ModuleName)
-		NewFunctionExporter().ExportFunctions(wasiBuilder)
+		wasiBuilder := r.NewHostModuleBuilder(wasi_snapshot_preview1.ModuleName)
```

Comment:
this is the only constant left exported, as it is needed when you use FunctionExporter.